mediacenter/views.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.admin.views.decorators import staff_member_required
+from django.core.urlresolvers import reverse_lazy
+from django.views.generic import (CreateView, DeleteView, DetailView, ListView,
+                                  UpdateView)
+
+from .models import Document
+
+
+class Index(ListView):
+    model = Document
+    template_name = 'mediacenter/index.html'
```

Comment:
everywhere else you rely on the name computed by the django generic views, why not here ?